wp-content/themes/humanity-theme/patterns/taxonomy-filters.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+/**
+ * Title: Taxonomy filters pattern
+ * Description: Taxonomy filters pattern for the theme
+ * Slug: amnesty/taxonomy-filters
+ * Inserter: no
+ */
+
+?>
+
+<!-- wp:group {"tagName":"div","className":"taxonomyArchive-filters"} -->
+<div class="wp-block-group taxonomyArchive-filters">
+<!-- wp:pattern {"slug":"amnesty/custom-select"} /-->
```

Comment:
I can't locate this pattern; could you point me to it, please?

Reply:
The pattern mentioned and the file it is included in are unused patterns so I have removed them from the PR. I must have missed them before PR'ing